src/ServerSide/InternalTypes.elm

```diff
@@ -75,6 +75,9 @@ decodeNodeType =
                     "node" ->
                         Json.Decode.map NodeEntry (decodeNode)
 
+                    "tagger" ->
+                        Json.Decode.map NodeEntry (decodeTagger)
```

Comment:
`decodeTagger` uses `decodeNode` which is recursive. I added a test and it works.